docs/source/upcoming_release_notes/issue_number-add_upper_limit_threshold_to_vcn.rst

```diff
@@ -0,0 +1,30 @@
+issue_number add upper limit threshold to vcn
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```

Comment:
The additional component itself looks fine, but you need to update the release notes to add yourself as a contributor and note what changed.